examples/tag_reader.rs

```diff
@@ -28,6 +29,12 @@ fn main() {
 	println!("Album: {}", tag.album().unwrap_or("None"));
 	println!("Genre: {}", tag.genre().unwrap_or("None"));
 
+	// import keys from https://docs.rs/lofty/latest/lofty/enum.ItemKey.html
+	println!(
+		"Album Artist: {}",
+		tag.get_string(&AlbumArtist).unwrap_or("None")
```

Comment:
Thanks! Could use you the full path here instead of importing the variant, just to be extra clear? 🙂